### PR TITLE
chore: fix linting issues

### PR DESCRIPTION
**Pull request by katcipis**
# Reason for This Change

Linting with Go 1.19 shows some new issues, nothing major.

### CODE REVIEW

File: errors/error.go [hunk 1]
```diff
@@ -70,41 +71,41 @@ const separator = ": "
...
-//		Only the first hcl.Diagnostic will be used.
-//		If hcl.Range is not set, the diagnostic subject range is pulled.
-//		If the string Description is not set, the diagnostic detail field is
-//		pulled.
-//	hcl.Diagnostic
-//		Same behavior as hcl.Diagnostics but for a single diagnostic.
-//	string
-//		The error description. It supports formatting using the Go's fmt verbs
-//		as long as the arguments are not one of the defined types.
+//		errors.Kind
```
Comment (katcipis) [inline errors/error.go:74]: I think this fmt fix is related to Go 1.19 additions on Go doc, now supporting links/lists/etc.

File: hcl/eval/eval.go [hunk 2]
```diff
@@ -155,7 +154,7 @@ func TokensForExpression(expr hhcl.Expression) (hclwrite.Tokens, error) {
 		exprdata = []byte(fnameBytes[len(injectedTokensPrefix()):])
 	} else {
 		var err error
-		exprdata, err = ioutil.ReadFile(string(fnameBytes))
+		exprdata, err = os.ReadFile(string(fnameBytes))
```
Comment (i4ki) [inline hcl/eval/eval.go:157]: hmm yeah got this here as well. They were already in my branch, but yeah better to fix this separately.

**Review by i4ki — APPROVED** [review]
lgtm but it seems gofmt windows is giving a different result than the one in linux o.o

File: cmd/terramate/e2etests/runner_windows_test.go [hunk 3]
```diff
@@ -1,51 +1,51 @@
-// Copyright 2022 Mineiros GmbH
```
[the remark this answers is not in the document]
Comment (katcipis) [reply cmd/terramate/e2etests/runner_windows_test.go:1]: this is the diff produced by running go fmt, no idea why it got so messed up :-(

**Pull request merged**